.github/workflows/docker.yml

```diff
@@ -1,9 +1,6 @@
 name: docker
 
-on:
-  push:
-    tags:
-    - '*'
+on: [push, workflow_dispatch]
```

Comment:
can you revert this part? this repo we do use tags

Reply:
done